tagging.md

```diff
@@ -34,384 +31,209 @@ Advanced-users application might let you put any tag in your file. But for the r
 We also need an official list simply for developpers to be able to display relevant information in their own design (if they choose to support a list of meta-information they SHOULD know which tag has the wanted meaning so that other apps could understand the same meaning).
 
 ## Tag translations
+
 To be able to save tags from other systems to Matroska we need to translate them to our system. There is a translation table [on our site](othertagsystems/comparetable.html).
 
 ## Tag Formatting
 
 * The TagName SHOULD always be written in all capital letters and contain no space.
-* The fields with dates SHOULD have the following format: YYYY-MM-DD
-    HH:MM:SS.MSS YYYY = Year, -MM = Month, -DD = Days, HH = Hours, :MM = Minutes,
-    :SS = Seconds, :MSS = Milliseconds. To store less accuracy, you remove items
-    starting from the right. To store only the year, you would use, "2004".
-    To store a specific day such as May 1st, 2003, you would use "2003-05-01".
-  
-* Fields that require a Float SHOULD use the "."
-    mark instead of the "," mark. To display it differently for another
-    local, applications SHOULD support auto replacement on display. Also, a thousandths
-    separator SHOULD NOT be used.
-* For currency amounts, there SHOULD only be a numeric value in the Tag. Only
-    numbers, no letters or symbols other than ".". For instance, you
-    would store "15.59" instead of "$15.59USD".
+* The fields with dates SHOULD have the following format: YYYY-MM-DD HH:MM:SS.MSS YYYY = Year, -MM = Month, -DD = Days, HH = Hours, :MM = Minutes, :SS = Seconds, :MSS = Milliseconds. To store less accuracy, you remove items starting from the right. To store only the year, you would use, "2004". To store a specific day such as May 1st, 2003, you would use "2003-05-01".
```

Comment:
I would say `MM = month` etc. instead of `-MM = Month` etc.
(i.e. without the leading sign and I don’t see a need for capitals here)